src/test/java/com/suse/netapi/client/SaltStackClientMockTests.java

```diff
@@ -0,0 +1,111 @@
+package com.suse.saltstack.netapi.client;
```

Comment:
The package is not matching the folder structure, please fix the folder structure in this case!